### ISSUE TITLE
Add an h1 heading to search results page/s

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1747]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Search results pages (both grouped and "all")

### Description of error / warning

Missing first level heading

### WCAG success criterion

[1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)

### WCAG conformance level

A

### Recommended fix

Add an h1 heading to the page

### Tool used to find the problem

wave

### Screenshot

<img width="2104" height="792" alt="Image" src="https://github.com/user-attachments/assets/a67baa7c-c937-4d04-986b-ac4c90995658" />
